src/api/libcellml/component.h

```diff
@@ -131,7 +131,7 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
      *
      * @param variable The variable to add.
      */
-    void addVariable(const VariablePtr &variable);
+    bool addVariable(const VariablePtr &variable);
```

Comment:
Missing documentation for return statement.

Reply:
Done in 249c836e.